test/codec-config.js

```diff
@@ -0,0 +1,78 @@
+const test = require('brittle')
+const ffmpeg = require('..')
+
+test('getSupportedConfig for pixel formats', (t) => {
+  const codecCtx = new ffmpeg.CodecContext(ffmpeg.Codec.AV1.encoder)
+  using pixFormats = codecCtx.getSupportedConfig(
+    ffmpeg.constants.codecConfig.PIX_FORMAT
+  )
+
+  t.ok(pixFormats instanceof Int32Array, 'returns Int32Array')
+  t.ok(pixFormats.length > 0, 'has pixel formats')
+})
```

Comment:
Sorry I introduce an error in my snippet.

```suggestion
test('getSupportedConfig for pixel formats', (t) => {
  using codecCtx = new ffmpeg.CodecContext(ffmpeg.Codec.AV1.encoder)
  const pixFormats = codecCtx.getSupportedConfig(
    ffmpeg.constants.codecConfig.PIX_FORMAT
  )

  t.ok(pixFormats instanceof Int32Array, 'returns Int32Array')
  t.ok(pixFormats.length > 0, 'has pixel formats')
})
```

We should do the same for other test cases.